package.json

```diff
@@ -42,32 +42,28 @@
 		"workerize-loader": "^1.3.0"
 	},
 	"dependencies": {
-		"@babel/core": "^7.10.3",
-		"@babel/plugin-proposal-object-rest-spread": "^7.10.3",
+		"@babel/core": "^7.11.0",
 		"@babel/plugin-transform-react-jsx": "^7.10.3",
-		"@babel/polyfill": "^7.10.1",
-		"@babel/preset-env": "^7.10.3",
-		"@babel/preset-stage-0": "^7.8.3",
+		"@babel/preset-env": "^7.11.0",
 		"babel-loader": "^8.1.0",
 		"babel-plugin-istanbul": "^6.0.0",
 		"chalk": "^2.3.0",
+		"core-js": "^3.6.5",
 		"dlv": "^1.1.3",
 		"errorstacks": "^1.3.0",
 		"expect": "^24.9.0",
-		"jasmine-core": "^3.5.0",
```

Comment:
`karma-jasmine` already includes `jasmine-core` as a dep so I don't think we need our own dep here.